### PR TITLE
Allow admins to delete/archive proposals

### PR DESCRIPTION
**Pull request by samuveth**
- Also changed the name of the loading state, in preparation for more items

### CODE REVIEW

File: src/views/Proposal.vue [hunk 1]
```diff
@@ -259,6 +259,12 @@ export default {
     },
     symbols() {
       return this.space.strategies.map(strategy => strategy.params.symbol);
+    },
+    isAdmin() {
+      return (
+        this.proposal.address === this.web3.account ||
+        this.space.admins.includes(this.web3.account)
```
Comment (bonustrack) [inline src/views/Proposal.vue:266]: Can you make this not case sensitive? Something like this:
```
const admins = (this.space.admins || []).map(admin => admin.toLowerCase());
if (admins.includes(this.web3.account.toLowercase()) ...
```

**Review by ChaituVR — COMMENTED** [review]
Not related to current PR changes,

but when I click on the loading icon, I am able to open the delete dropdown, does it look fine?
3rq23423423

**Comment by bonustrack** [comment]
@ChaituVR Good catch, this should not happen

**Comment by samuveth** [comment]
I think this PR and the hub PR are still missing the case where the owner of the space want to delete archive a proposal. Or will the owner be in admins by default?

**Comment by ChaituVR** [comment]
isCreator/space creator is not allowed to archive proposal (i think).. I've added it in the hub but @bonustrack asked me to remove it

**Comment by samuveth** [comment]
> isCreator/space creator is not allowed to archive proposal (i think).. I've added it in the hub but @bonustrack asked me to remove it

Yes that's right, thanks for letting me know.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**